submissions/Vlada-khom/html-css-popup/styles.css

```diff
@@ -0,0 +1,185 @@
+body {
+  background-color: rgba(0, 255, 255, 0.2);
+}
+
+header nav {
+  text-align: right;
+  font-family: "Roboto";
+}
+header nav .navLink {
+  vertical-align: middle;
+  display: inline-block;
+  padding: 10px;
+  text-decoration: none;
+  color: black;
+}
+header nav .navLink .imgLink {
+  width: 30px;
+  height: 30px;
+}
+header nav .navLink .avatar {
+  border-radius: 50%;
+}
+header nav .burger {
+  position: relative;
+}
+header nav .burger .input {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  clip: rect(0, 0, 0, 0);
+}
+header nav .burger #burgerCheckbox:checked ~ .main {
+  display: flex;
+}
+header nav .navLink:focus {
+  outline: 2px solid blue;
+}
+
+.popupContainer {
+  width: 350px;
+  height: 545px;
+  border: 1px solid grey;
+  display: none;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: 30px 5px 0px 5px;
+  box-shadow: 0 0 15px grey;
+  position: absolute;
+  right: -26px;
+  top: 57px;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+.popupContainer .containerForItem {
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+.popupContainer .visibleBlock {
+  display: flex;
+}
+.popupContainer .itemBlock {
+  padding: 5px;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+.popupContainer .itemBlock .itemPopup {
+  width: 70px;
+  padding: 15px;
+  font-family: "Roboto";
+  text-decoration: none;
+  color: black;
+}
+.popupContainer .itemBlock .itemPopup .imgBlock {
+  width: 70px;
+  height: 65px;
+}
+.popupContainer .itemBlock .itemPopup .imgBlock img {
+  max-width: 50px;
+  max-height: 50px;
+}
+.popupContainer .itemBlock .itemName {
+  font-family: "Roboto";
+}
+.popupContainer .itemPopup:focus {
+  outline: 2px solid blue;
+}
+
+.popupContainer:nth-child(2):before {
+  content: "";
+  display: block;
+  width: 20px;
+  height: 20px;
+  border-bottom: 1px solid grey;
+  border-left: 1px solid grey;
+  transform: rotate(135deg);
+  position: absolute;
+  left: 253px;
+  top: -12px;
+  background: linear-gradient(to top right, #ccffff, #ccffff, #ccffff);
```

Comment:
In some cases, you use `rgba` or `hex` color scheme in the other embedded in browser colors. Try to use no more than two color schemas in your project.